src/app/utils/mail.ts

```diff
@@ -98,14 +99,16 @@ export const generateAutoreplyPdf = async (
 
 type AutoreplyHtmlData = {
   autoReplyBody: string[]
-} & (AutoreplySummaryRenderData | {})
+} & (AutoreplySummaryRenderData | Record<string, unknown>)
```

Comment:
could you explain this?

Reply:
Fixed, thanks for calling me out :(

`{}` is now not allowed, so I tried to take the easy way out. Duplicated the shape instead.

Reply:
LOL I wasn't aware that there was anything to call out, but glad it worked out anyway